extrabees/src/main/java/binnie/extrabees/worldgen/WorldGenHiveMarble.java

```diff
@@ -31,19 +31,18 @@ public boolean generate(World world, Random rand, BlockPos position) {
 			return false;
 		}
 
-		//generate when one face is different from marble
+		//generate when at least two faces are different from marble
 		int otherFace = 0;
 		for (EnumFacing face : EnumFacing.values()) {
 			if (!world.getBlockState(position.offset(face)).getBlock().equals(blockAtPos)) {
 				otherFace++;
 				if (otherFace > 1) {
```

Comment:
otherFace is only > 1 when the if condition above triggered a second time, therefore i changed the comment above to match the already implemented behaviour.